styleguide/_themes/global/scss/modifiers.scss

```diff
@@ -76,6 +76,18 @@ img { max-width: 100%; }
   margin: 0 auto;
 }
 
+// Horizontal rules
+hr {
+  border: 0;
+  border-top: 1px solid $gray-lighter;
+  margin-bottom: $line-height-computed / 2;
+  margin-top:    $line-height-computed / 2;
+
+  &.red {
```

Comment:
Since this is new, do we currently have any `<hr class="red">` in use anywhere?   Because it seems like we could easily just have a `hr-red` class that could live separately?